doc/manual/oqum/hazard/01d_config_classical.tex

```diff
@@ -146,19 +146,32 @@
 
 \begin{minted}[firstline=1,linenos=true,firstnumber=23,fontsize=\footnotesize,frame=single,bgcolor=lightgray]{ini}
 [site_params]
-site_model_file = site_model.xml
+site_model_file = site_model.csv
 \end{minted}
 
 The file containing the site model has the following structure:
 
 \begin{listing}[htbp]
-  \inputminted[firstline=1,firstnumber=1,fontsize=\footnotesize,frame=single,linenos,bgcolor=lightgray]{xml}{oqum/hazard/verbatim/input_site_model.xml}
+  \inputminted[firstline=1,firstnumber=1,fontsize=\footnotesize,frame=single,linenos,bgcolor=lightgray]{xml}{oqum/hazard/verbatim/input_site_model.csv}
   \caption{Example site model input file}
   \label{lst:input_site_model}
 \end{listing}
 
+Notice that the $0$ for the field \texttt{vs30measured} means that the
+\texttt{vs30} field is inferred, not measured. This is the default, so you
+can just skip the field. Also, most of the GMPEs are not sensitive to
+it.  The same can be said for the \texttt{backarc} parameter: $0$
+means false and this is the default, so you can skip such column. All
+columns that are not needed by the GMPEs your are using can be safely
+skipped, while you will get an error if a relevant column is missing.
+
 If the closest available site with soil conditions is at a distance greater than 5~km from the investigation location, a warning is generated.
 
+\textbf{Note:} For backward-compatibility reasons, the site model file can
+also be given in XML format. That old format is deprecated but there are
```

Comment:
Should we still keep the xml format documented for legacy users?

Reply:
I would not do that. Legacy users will already have their way to produce site model files in the old format and that will keep working. On the other hand, we do not want to give bad ideas to the new users.